go.mod

```diff
@@ -131,3 +132,5 @@ replace (
 	// the following version across all dependencies.
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 )
+
+replace github.com/tendermint/tendermint => github.com/yihuang/tendermint v0.34.14-0.20220518061324-81344ac9464d
```

Comment:
upstream PR: https://github.com/tendermint/tendermint/pull/8574

Reply:
@yihuang 
Apologies for intruding into this wonderful space, but do you think `--delete-pending-block` is still needed in the upstream `cosmos-sdk` ? (AFAIK, it did not go into the upstream https://github.com/cosmos/cosmos-sdk/commit/51d2de582d036ece4af10267e889f6fdd97f8acf )

The reason I'm asking is because it was highly useful for me when I cherry-picked it to allow akash rollback work (had to use that `--delete-pending-block` flag which was very helpful). :-) 

More details in
https://github.com/cometbft/cometbft/pull/1610